man/ocamlrun.m

```diff
@@ -145,9 +161,21 @@ Turn on randomization of all hash tables by default (see the
 module of the standard library). This option takes no
 argument.
 .TP
+.BR H
+Allocates heap chunks by mmapping huge pages. Huge pages are locked into
+memory, and are not swapped.
+.TP
 .BR h
 The initial size of the major heap (in words).
 .TP
+.BR w
```

Comment:
```suggestion
.B t
Sets the trace level for the debug runtime (ignored otherwise).
.TP
.BR w
```
and then the comment way at L138 (line `\" FIXME missing: c, H, t, w, W see MPR#7870`) can be deleted